DataFormats/Parameters/include/DataFormatsParameters/GRPECSObject.h

```diff
@@ -109,6 +110,18 @@ class GRPECSObject
   /// same with comma-separate list of detector names
   DetID::mask_t getDetsReadOut(const std::string& only, const std::string& skip = "") const { return getDetsReadOut(DetID::getMask(only), DetID::getMask(skip)); }
 
+  // methods to manipulate the list of FLPs in the run
+  std::vector<unsigned short> getListOfFLPs() const { return mFLPs; }
```

Comment:
should be 
`const std::vector<unsigned short>& getListOfFLPs() const { return mFLPs; }`

Reply:
Sure, will change it ASAP.

Reply:
done.